libs/math/tests/unit/distance/entropy_tests.cpp

```diff
@@ -37,10 +37,10 @@ TYPED_TEST(EntropyTest, entropy)
 
   ArrayType A(4);
 
-  A.Set(SizeType{0}, DataType(0.1));
-  A.Set(SizeType{1}, DataType(0.2));
-  A.Set(SizeType{2}, DataType(0.3));
-  A.Set(SizeType{3}, DataType(0.4));
+  A.Set(SizeType{0}, fetch::math::Type<DataType>("0.1"));
+  A.Set(SizeType{1}, fetch::math::Type<DataType>("0.2"));
+  A.Set(SizeType{2}, fetch::math::Type<DataType>("0.3"));
+  A.Set(SizeType{3}, fetch::math::Type<DataType>("0.4"));
 
   EXPECT_NEAR(double(statistics::Entropy(A)), 1.84643934467102, 1e-4);
```

Comment:
(optional) As far as we use fixed point here, may be it can be EXPECT_EQ?

Reply:
i think this is tested for floats as well though.
we can replace 1e-4 with function tolerance though, right?